### PR TITLE
Updated iOS & JS chat quick start

### PR DESCRIPTION
**Pull request by tensiuyan**
I have updated the iOS and the JS chat quick start to match the general quick start. They r steps for adding Skygear chat to their app.

### CODE REVIEW

**Comment by tensiuyan** [comment]
Duplicate with #479

**Comment by tensiuyan** [comment]
@chpapa could you please help take a look at this Pull Request? The update is mainly about updating the Chat Quick Start to match the updated General Quick Start.

See if you have any comments? I plan to have half day to work on the doc today. So great if my previous pull request can be merged first :) thanks

File: content/chat/quick-start/js.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+```javascript
+<script src ="http://code.skygear.io/js/chat/latest/skygear-chat.min.js" </script>
+```
+
+### Method 2: Npm Install
+
+1. Add Skygear Chat to your project using npm install.
+```bash
+npm install skygear-chat
```
Comment (chpapa) [inline content/chat/quick-start/js.md]: better use `npm install --save skygear-chat`?

File: content/chat/quick-start/js.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+![Skygear plug-ins](/assets/common/enable-chat-plugin-on-portal.png)
+
+Cool your are all set now.
+
+## What's next from here
+
+You got the basic. Next, learn more about:
+* [Skygear Chat](https://docs.skygear.io/guides/chat/basics/js/) or [API reference](https://doc.esdoc.org/github.com/skygeario/chat-SDK-JS/class/lib/index.js~SkygearChatContainer.html)
+* [Skygear User Authentication](https://docs.skygear.io/guides/auth/basics/js/)
```
Comment (chpapa) [inline content/chat/quick-start/js.md]: Add these demo project?
* https://github.com/skygear-demo/ionic-chat-demo
* https://github.com/skygear-demo/react-chat-demo

File: content/chat/quick-start/ios.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+![Skygear plug-ins](/assets/common/enable-chat-plugin-on-portal.png)
+
+Cool your are all set now.
+
+## What's next from here
+
+You got the basic. Next, learn more about:
+* [Skygear Chat](https://docs.skygear.io/guides/chat/basics/ios/)
+* [Skygear User Authentication](https://docs.skygear.io/guides/auth/basics/ios/)
```
Comment (chpapa) [inline content/chat/quick-start/ios.md]: * Add "Checkout iOS UIKit README for common UI elements for chat using with Skygear Chat" (sth like that ~~~) https://github.com/SkygearIO/chat-SDK-iOS/blob/master/docs/UIKit/index.md
* Add link to demo project and tutorial? https://github.com/skygear-demo/swift-chat-uikit-demo
https://github.com/skygear-demo/swift-chat-demo

**Comment by tensiuyan** [comment]
@chpapa updated see if they r good?

**Pull request closed without merging**